pennylane/measurements/__init__.py

```diff
@@ -157,6 +157,23 @@ def circuit(x):
     functions will return an instance of the corresponding measurement process
     (e.g. :class:`CountsMP`). This decision is just for design purposes.
 
+.. details::
+    :title: Serialization and Pytree format
+    :href: serialization
+
+    PennyLane measurements are automatically registered as `Pytrees <https://jax.readthedocs.io/en/latest/pytrees.html>`_ .
+
+    The :class:`~.MeasurementProcess` definitions are sufficient for all PL measurements.
```

Comment:
So does this mean that users creating custom measurements do not need to do anything extra?

Reply:
Oops. Actually that comment is wrong. They need to be overridden if the measurement process has extra metadata.

Reply:
I'll fix that in the tape pytree PR.